system/mp/test/mptest.cpp

```diff
@@ -727,7 +727,7 @@ int main(int argc, char* argv[])
             FILE *fp = fopen(argfile, "r");
             if (fp == NULL)
             {
-                PrintLog("MPTest: Error cannot open file <%s>", argfile);
+                PrintLog("MPTest: Error cannot open file <%s>", argfile.str());
```

Comment:
I would have thought argfile should be a char* that contains either NULL or argv[3] - why is it copying the contents?